### ISSUE TITLE
Where is check_sgp4 used?

### ISSUE THREAD

**Issue by hornig** [aerospaceresearch/orbitdeterminator#207]
https://github.com/aerospaceresearch/orbitdeterminator/blob/d312577a969b1aa8ad1bbd317d80ded42613de19/orbitdeterminator/tests/check_sgp4.py#L1-L67

I could not find it. at least to test .py used it (or I oversaw it). And it is not its own test. because pytest is not included and the filename does not have correct "test_" start or "test" ending.

**Comment by micksharma** [comment]
Hi @hornig  ,
I want to work on this issue ?

**Comment by hornig** [comment]
> Hi @hornig ,
> I want to work on this issue ?

yes, everyone can. just do it and if you have a solution commit a Pull-Request.
no need to ask here :)
